packages/web-forms/src/components/FormHeader.vue

```diff
@@ -75,20 +78,33 @@ const handleLanguageChange = (event: FormLanguage) => {
 
 		<div class="form-options">
 			<!-- if Form is not multilingual then we always show print button -->
-			<Button v-if="languages.length === 0" class="print-button" severity="secondary" rounded icon="icon-local_printshop" @click="print" />
+			<Button v-if="languages.length === 0" class="print-button" severity="secondary" rounded @click="print">
+				<IconSVG name="mdiPrinter" />
+			</Button>
 
 			<!-- show either hamburger or (print button and language changer) based on container size -->
 			<div v-else class="multilingual">
-				<Button icon="icon-menu" class="btn-menu" text rounded aria-label="Menu" @click="menu?.toggle" />
-				<Menu id="overlay_menu" ref="menu" :model="items" :popup="true" />
+				<Button class="btn-menu" text rounded aria-label="Menu" @click="menu?.toggle">
+					<IconSVG name="mdiMenu" />
+				</Button>
+				<Menu id="overlay_menu" ref="menu" :model="items" :popup="true">
+					<template #item="{ item }">
+						<a class="p-menu-item-link" v-if="item.command != null" @click="(event) => item.command?.({ originalEvent: event, item })">
```

Comment:
TypeScript static check doesn't understand that the `item.command` is already validated in the `v-if="item.command != null"` 🤦🏽‍♀️ That's why the _optional chaining operator_

Reply:
Looks like https://github.com/vuejs/vetur/issues/1212?

Maybe `!` instead of `?` is slightly more explicit since you know it isn't null at that point but not a big deal.

Reply:
There's also a lint warning here.

Reply:
Interesting, the `!` didn't fix the TypeScript static check and requires a bit more to make it work. Ultimately, I'm checking `item.command` is defined before calling it.

I've fixed the GH's lint warning.